.github/workflows/test-on-push.yml

```diff
@@ -1,5 +1,11 @@
 name: Tests
-on: [ push ]
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
```

Comment:
It looks good :+1:   
Could you please by the way remove from the same file whole `test-dco-sign-off` section with it's sub items?  
We validate DCO on PR creation level anyways and other ways seem to be unnecessary and buggy (like in case of this PR)

Reply:
Done!